CONTRIBUTING.asciidoc

```diff
@@ -89,19 +89,38 @@ Checkers
 qutebrowser uses http://tox.readthedocs.org/en/latest/[tox] to run its
 unittests and several linters/checkers.
 
-Currently, the following tools will be invoked when you run `tox`:
-
-* Unit tests using https://www.pytest.org[pytest].
-* https://pypi.python.org/pypi/pyflakes[pyflakes] via https://pypi.python.org/pypi/pytest-flakes[pytest-flakes]
-* https://pypi.python.org/pypi/pep8[pep8] via https://pypi.python.org/pypi/pytest-pep8[pytest-pep8]
-* https://pypi.python.org/pypi/mccabe[mccabe] via https://pypi.python.org/pypi/pytest-mccabe[pytest-mccabe]
-* https://github.com/GreenSteam/pep257/[pep257]
-* http://pylint.org/[pylint]
-* https://pypi.python.org/pypi/pyroma/[pyroma]
-* https://github.com/mgedmin/check-manifest[check-manifest]
-* `scripts/misc_checks.py` which checks for the following things:
+Currently, following tox environments are available:
+
+* Unit tests using https://www.pytest.org[pytest]:
```

Comment:
I know it's my text, but I think "Tests using ..." would make more sense, as they aren't only unittests.